alphadia/constants/default.yaml

```diff
@@ -41,7 +41,7 @@ library_prediction:
   # maximum charge state for predicted fragments
   max_fragment_charge: 2
   instrument: Lumos
-  checkpoint_folder_path: None
+  checkpoint_folder_path: null
```

Comment:
could you add a short comment here: what behaviour does 'null' mean, why would I set this?

also: really `null`?

Reply:
It's stupid, I agree. `null` in a yaml translates into `None` in python. `None` gets parsed as `'None'` 😭

Reply:
again what learned ..
could we change this to 
`checkpoint_folder_path: # empty: use default ~/peptdeep`

Reply:
Yep, thats the current behavior with `null`.
We couuuld, but in yaml `key: ` gets parsed as `key: null`.  😄 so maybe the verbose version is better?